Makefile

```
@@ -12,7 +12,8 @@ test-fast:
 	pytest -x --cov samtranslator --cov-report term-missing --cov-fail-under 95 -n auto tests/
 
 test-cov-report:
-	pytest --cov samtranslator --cov-report term-missing --cov-report html --cov-fail-under 95 tests/
+	pytest --cov samtranslator --cov-report term-missing --cov-report html --cov-fail-under 95 -n auto tests/
+	open htmlcov/index.html
```

Only on macOS by default AFAIK:

```suggestion
	open htmlcov/index.html &> /dev/null || true
```